app/src/main/assets/json/onboarding.json

```diff
@@ -18,6 +18,62 @@
           "type"         : "permissions"
         }
       ]
+    },
+    {
+      "onboardingType" : "completion",
+      "steps": [
+        {
+          "identifier": "testerOrParticipant",
+          "title": "I am a:",
+          "type": "dataGroups.singleChoiceText",
+          "items": [
+            {
+              "text": "Tester",
+              "value": "test_user"
+            },
+            {
+              "text": "Participant",
+              "value": ""
+            }
+          ],
+          "expectedAnswer": "test_user",
+          "skipIdentifier": "clinicChoices",
+          "optional": false
+        },
+        {
+          "identifier": "testerChoices",
```

Comment:
I may be mistaken, but I think this step may have just been for iOS team to help with their development.  The code that I will post today to the Android project will remove any need for "ux_tester" and so I think it's safe to remove this step.